transforms/transforms.js

```diff
@@ -1,4 +1,4 @@
-const quantum = require('quantum-js')
+const quantum = require('quantum-core')
```

Comment:
Fixes the issue where example codeblocks didn't render - it was due to using `quantum.select` from the old quantum which doesn't satisfy `looksLikeAnEntity`

Reply:
How did this require even work...

Reply:
Hexagon-js depends on quantum-js so the module was there.

The old selection api was similar enough not to break the new transformer :P